configs/machines/aleph.yaml

```diff
@@ -0,0 +1,124 @@
+# THE aleph.iccp.kr YAML config file
+
+name: aleph
+
+accounting: true
+batch_system: "pbs"
+
+operating_system: "SLES"
+
+jobtype: compute
+sh_interpreter: "/bin/sh"
+
+
+choose_jobtype:
+        post:
+                partition: serial
+        compute:
+                partition: iccp
+        tidy_and_resubmit:
+                partition: iccp
+
+choose_partition:
+        "iccp":
+                cores_per_node: 40
+        "serial":
+                cores_per_node: 1
+
+logical_cpus_per_core: 2
+
+threads_per_core: 1
+pool_dir: "/proj/awi"
+
+pool_directories:
+        pool: "/proj/awi"
+        projects: "/proj/awi"
+
+
+submitted: false
+hyper_flag: ""
+
+# standard environment setup
+#
+#
+
+useMPI: cray_mpich
+
+module_actions:
+        - "load ${PrgEnv}"
```

Comment:
@JanStreffing, what about starting with a `module purge` here? Maybe it is not necessary but at least it is a measure against conflicting module loads.